src/SourceGenerators/Uno.UI.SourceGenerators.netcore.Tests/Verifiers/XamlGenerator/CSGenerator.cs

```diff
@@ -71,7 +72,14 @@ public Test(XamlFile[] xamlFiles, string globalConfig = "")
 					TestState.AdditionalFiles.Add(($"/0/{xamlFile.FileName}", xamlFile.Contents));
 				}
 				TestState.AnalyzerConfigFiles.Add(("/.globalconfig", globalConfigBuilder.ToString()));
-				ReferenceAssemblies = ReferenceAssemblies.Default.AddPackages(ImmutableArray.Create(new PackageIdentity("Uno.UI", "4.4.20")));
+				
+				ReferenceAssemblies = new ReferenceAssemblies(
+						"net7.0",
```

Comment:
This will work also if we don't have net7.0 installed yet ? 

Also is it expected to remove the Uno.UI assembly?

Reply:
Uno is added through an overload below, and .NET 7 is required for this particular test.